libraries/cli/lib/bin.ts

```diff
@@ -122,11 +125,6 @@ async function main() {
     (<ListenerPlugin>plugin).setupEventListener();
   }
 
-  // Prepare modules
-  LOGGER.info("Preparing modules...");
-  await ModuleManager.instance.prepare();
```

Comment:
Why was the prepare removed?

Reply:
Woops, must have removed it accidentally. Fixed.